fcos-config.yaml

```diff
@@ -46,7 +46,33 @@ storage:
         name: core
       target: /usr/lib/systemd/user/podman.socket
       hard: false
+    - path: /home/core/.config/systemd/user/default.target.wants/cockpit-core-user.service
+      user:
+        name: core
+      group:
+        name: core
+      target: /home/core/.config/systemd/user/cockpit-core-user.service
+      hard: false
   files:
+      # Add systemd unit for the cockpit bearer token
```

Comment:
Fwiw, it's really not obvious what in the unit file is related to a 'bearer token'. I see a cockpit service being started as the core user, this is reflected in the name of the unit file `cockpit-core-user.service`. If this is key to connecting to cockpit without auth, maybe it's worth giving more details how this works in the PR?